gadget/internal/mkfs.go

```diff
@@ -32,16 +32,16 @@ import (
 type MkfsFunc func(imgFile, label, contentsRootDir string) error
 
 var (
-	mkfsHandlers = map[string]MkfsFunc{
-		"vfat": MkfsVfat,
-		"ext4": MkfsExt4,
+	MkfsHandlers = map[string]MkfsFunc{
```

Comment:
Maybe we could still do a little refactor here so that the call sites has no werid `""` being passed as the last argument:

```
// Mkfs creates a filesystem of given type and provided label in the device or file.
func Mkfs(typ, img, label string) error {
    return MkfsWithContent(typ, img, label, "")
}

// Mkfs creates a filesystem of given type and provided label in the device or file.
// The filesystem is populated with contents of contentRootDir.
func MkfsWithContent(typ, img, label, contentRootDir string) error {
    h, ok := mkfsHandlers[typ]
    ...
}
```